### PR TITLE
[feat](nereids) support create view in nereids

### PR DESCRIPTION
**Pull request by feiniaofeiafei**
This pr supports create view stmt in nereids planner. For example:
```sql
create view v1 as select * from t1
```

### CODE REVIEW

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run p0

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by 924060929** [comment]
add a udf test case? e.g.
```sql
use current_db;
create view test_view
as
select other_db.my_udf(col)
from other_db.tbl
```

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run feet

**Comment by feiniaofeiafei** [comment]
run feut

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/info/CreateViewInfo.java [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+        }
+        this.logicalQuery = logicalQuery;
+        this.querySql = querySql;
+        this.simpleColumnDefinitions = simpleColumnDefinitions;
+    }
+
+    /** init */
+    public void init(ConnectContext ctx) throws UserException {
+        analyzeAndFillRewriteSqlMap(querySql, ctx);
+        OutermostPlanFinder outermostPlanFinder = new OutermostPlanFinder();
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/info/CreateViewInfo.java:102]: could we use Singleton instance?

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/info/CreateViewInfo.java [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+ */
+public class CreateViewInfo {
+    private final boolean ifNotExists;
+    private final TableNameInfo viewName;
+    private final String comment;
+    private final LogicalPlan logicalQuery;
+    private final String querySql;
+    private final List<SimpleColumnDefinition> simpleColumnDefinitions;
+    private final List<Column> finalCols = Lists.newArrayList();
+    private final List<Slot> outputs = Lists.newArrayList();
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/info/CreateViewInfo.java:82]: outputs could be a local var?

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/info/CreateViewInfo.java (hunk 2 above)
Comment (feiniaofeiafei) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/info/CreateViewInfo.java]: Everything has been modified according to the comments

**Review by morrySnow — APPROVED** [review]

**Review by 924060929 — APPROVED** [review]

**Pull request merged**